concepts/courtroom.md

```diff
@@ -0,0 +1,58 @@
+# META Courtroom
+
+This document outlines the concept of a META courtroom which enable to
+collateralise service promises and enforce punitive measures.
+
+## Status of this Document
+
+The META protocol is at an early stage of development, expect this document to
+change frequently during this early phase.
+
+## Outline
+
+The META Courtroom aims to provide a framework for META services and service
+clients which is used for conflict resolution in relation to a specific
+service offerings and promises.
+The service provider specifies their service offering by tendering a process
+in which they can be held accountable for not complying with their quality guarantees.
+Provider swears to this promise by depositing a collateral which they stake to lose
+in case of foul play.
+
+If some service was not provided as promised, clients can open a case and start
+a litigation process. In case the evidence is available and can prove the accusation,
+the service provider can be held accountable by an automatic self-enforcing punitive
+measure.
+
+Such accountability is crucial in the design of scalable decentralised service economies.
+
+The META Courtroom concept is based on [swarm swap swear and swindle](http://swarm-gateways.net/bzz:/theswarm.eth/ethersphere/orange-papers/1/sw%5E3.pdf),
+a generalised framework for incentivised service provision.
+
+The courtroom framework will offer a generic interface for handling cases,
+register clients and services and submitting a new case where each service can use
+and set its own specific game rules.
+e.g which evidence types a client should submit for a case, grace periods for submitting
+evidences, the compensation amount for the case of a valid case...
+
+
+## Implementation
+
+The courtroom framework include a solidity smart contracts suite, an ABI interface
+specification and tools to interact with these contracts.
+
+A courtroom suite includes  abstract generic `Swear` contract which knows about the tendering and litigation process,
+i.e., how to open a case and conduct a trial.
+An abstract trial is described by a finite state automaton states of which correspond to stages of litigation and
```

Comment:
What about 
"by a finite state automaton which correspond to stages of litigation and..."
https://en.wikipedia.org/wiki/Automaton 
?